cockatrice/src/dlg_forgotpasswordchallenge.cpp

```diff
@@ -27,9 +27,9 @@ DlgForgotPasswordChallenge::DlgForgotPasswordChallenge(QWidget *parent) : QDialo
     }
 
     if (servers.getFPHostname().isEmpty() && servers.getFPPort().isEmpty() && servers.getFPPlayerName().isEmpty()) {
-        QMessageBox::warning(this, tr("Forgot Password Challenge Warning"),
-                             tr("Oops, looks like something has gone wrong.  Please restart the forgot password "
-                                "process by using the forgot password button on the connection screen."));
+        QMessageBox::warning(this, tr("Reset Password Challenge Issue"),
+                             tr("Oops, looks like something has gone wrong. Please restart the process "
```

Comment:
I think referencing a "process" might be intimidating to some users that might be less familiar in the language, instead just asking them to try again seems more fitting.

Reply:
Agreed. Fixing the repetition here is not enough. :)